src/app/events/events.page.ts

```diff
@@ -39,4 +47,17 @@ export class EventsPage implements OnInit {
       },
     });
   }
+
+  deleteEvents(event: any): void {
```

Comment:
why again "any" ? We have the models, right?

Reply:
fixed